From 618350acb244804522bba66aab485d4995527526 Mon Sep 17 00:00:00 2001
From: Luc Menut <lmenut@mageia.org>
Date: Mon, 8 Dec 2014 22:35:32 +0100
Subject: [PATCH 03/16] Enable PrimaryPaste

---
 xsettings-kde.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/xsettings-kde.c b/xsettings-kde.c
index 6162136..d6dbb2f 100644
--- a/xsettings-kde.c
+++ b/xsettings-kde.c
@@ -530,6 +530,8 @@ void initial_init () {
 		xsettings_manager_set_string(managers[i], "Net/FallbackIconTheme", "gnome");
        /* KDE always shows menu images, so make sure GTK+ does, too */
         xsettings_manager_set_int(managers[i], "Gtk/MenuImages", 1);
+		/* KDE always pastes clipboard content on middle-click */
+		xsettings_manager_set_int(managers[i], "Gtk/EnablePrimaryPaste", 1);
       if (vgtkimm)
             xsettings_manager_set_string (managers[i], "Gtk/IMModule", vgtkimm);
 		xsettings_manager_notify (managers[i]);
-- 
1.9.5

